src/com/teamcautionrobotics/robot2018/Climb.java

```diff
@@ -0,0 +1,25 @@
+package com.teamcautionrobotics.robot2018;
+
+import edu.wpi.first.wpilibj.VictorSP;
+
+public class Climb {
+
+    private VictorSP climbMotor;
+
+    public Climb(int motorPort) {
+        climbMotor = new VictorSP(motorPort);
+    }
+
+    public void move(double power) {
+        climbMotor.set(power);
+    }
+
+    public void ascend() {
+        climbMotor.set(1.0);
```

Comment:
Call `this.move` instead of the motor directly (for both ascend and descend). This way, changes relating to the motor are in one place instead of three.